<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="FIRST_RB">
     <property name="text">
      <string>First 100 records</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="LAST_RB">
     <property name="text">
      <string>Last 100 records</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="YEAR_RB">
     <property name="text">
      <string>Selected year</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="YEAR_FR">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="YEAR_LB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Choose the year</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="YEAR_CB">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="PERIOD_RB">
     <property name="text">
      <string>Selected period</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="PERIOD_FR">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="START_LB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>From day</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="FROMDAY">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="END_LB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>To day</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="TODAY">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="ALL_RB">
     <property name="text">
      <string>All records</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
